contrib/backporting/common.sh

```diff
@@ -19,10 +19,10 @@ set -e
 get_remote () {
   local remote
   remote=$(git remote -v | \
-    grep "git@github.com:cilium/cilium.git\|https://github.com/cilium/cilium" | \
+    grep "github.com[/:]cilium/cilium" | \
```

Comment:
/cc I think @brb introduced this, not sure exactly which remotes he had trouble with that led to the current regex here.